packages/dev/core/src/Misc/sceneSerializer.ts

```diff
@@ -391,7 +411,7 @@ export class SceneSerializer {
             for (let i = 0; i < toSerialize.length; ++i) {
                 if (withChildren) {
                     toSerialize[i].getDescendants().forEach((node: Node) => {
-                        if (node instanceof Mesh && toSerialize.indexOf(node) < 0 && !node.doNotSerialize) {
+                        if (toSerialize.indexOf(node) < 0 && !(node as any).doNotSerialize) {

```

Comment:
nit - why are you casting to any? Node has `doNotSerialize`. Should it also support skeleton/material?

Reply:
Oh you're right, not sure why I was seeing type warnings before o.o